### PR TITLE
add category item in menu

### LINKED ISSUE #2: [US-1] List of Categories

### PR DESCRIPTION
**Pull request by suademre**
add category item in Menu.
i wrote short testing. 
add styling.
i changed the daten
I separated the css files.
This resolves #2

### CODE REVIEW

File: src/components/Nav/index.js [hunk 1]
```diff
@@ -26,61 +36,14 @@ function Nav() {
                 </CloseButton>
 
                 <MenuList>
-                    <MenuItem>Categories</MenuItem>
+                    <MenuTittle>Categories</MenuTittle>
```
Comment (jamarob) [inline src/components/Nav/index.js:39]: Only li are allowed to as direct children of ul. While tittle sounds cute I think you meant title ;)

File: src/components/Nav/index.js [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
 
+import categories from '../../data/category.json';
+import MenuItem from 'components/Nav/MenuItem';
+import {
+    HeaderNav,
+    SidebarSection,
+    CloseButton,
+    MenuList,
+    MenuTittle,
+} from './styles';
```
Comment (jamarob) [inline src/components/Nav/index.js:13]: Not really sure about the distinction between styles and components.. components have styles, too. Also `SidebarSection` has some custom logic.

File: src/components/Nav/styles.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import styled, {css} from 'styled-components/macro';
+
+export const HeaderNav = styled.nav`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-sizing: border-box;
```
Comment (jamarob) [inline src/components/Nav/styles.js:7]: Imho you want this on all elements to save you some headache in the future, think about adding it to a GlobalStyle

File: src/components/Nav/styles.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    z-index: 100;
+`;
+export const SidebarSection = styled.section`
+    position: fixed;
+    top: 0;
+    width: 70vw;
+    height: 100vh;
+    background-color: #f9f9f9;
+    z-index: 101;
+    transition: left 0.4sn ease;
```
Comment (jamarob) [inline src/components/Nav/styles.js:23]: Typo, or what does `sn` do?

**Review by jamarob — COMMENTED** [review]
gj 👍 left some comments

Some general suggestions:
- use better commit messages, second and third don't tell my anything about what you did. Commit messages are a nice way to document your work.
- prefer layout components over letting components position and layout themself, saves you a lot of css to write
- you set a lot very specific paddings and margins, save yourself some headache and use a global style with css vars for sizes. This gives a more uniform look and saves you from pushing pixels

File: src/components/Nav/index.js (hunk 2 above)
Comment (suademre) [reply src/components/Nav/index.js:13]: i just seperated. i would like to try it

File: src/components/Nav/styles.js (hunk 4 above)
Comment (suademre) [reply src/components/Nav/styles.js:23]: i changed

**Pull request merged**